app/views/build_estimates/vehicle_value.html.slim

```diff
@@ -0,0 +1,26 @@
+= link_to t("generic.back"), previous_wizard_path, class: "govuk-back-link"
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    span.govuk-caption-l
+      = t(".caption")
+    h1.govuk-heading-l
+      = t(".heading")
+
+    = form_for(@form,
+                 url: wizard_path,
+                 method: :put) do |form|
+
+      = form.govuk_error_summary t("generic.error_summary_title")
+
+      = render "money_input",
```

Comment:
You're going to hate this one... I'm getting flaky behaviour on this input. _every so often_ when I update the value in the field and click 'Save and continue' then click back, the updated value is lost and the previous value is retained. See screen capture:
https://user-images.githubusercontent.com/113888736/193307375-5b986188-0693-4996-948f-c9b46c4c6b0a.mov

Reply:
> You're going to hate this one... I'm getting flaky behaviour on this input. _every so often_ when I update the value in the field and click 'Save and continue' then click back, the updated value is lost and the previous value is retained. See screen capture: https://user-images.githubusercontent.com/113888736/193307375-5b986188-0693-4996-948f-c9b46c4c6b0a.mov

I've converted the input into a 'pure numeric' as per the style guide. It's not quite clear to me what we should be using https://govuk-form-builder.netlify.app/form-elements/text-input/ - but it seems to hint that numeric_input shouldn't be used